tools/sanity_checks.py

```diff
@@ -53,10 +53,17 @@ def setUpClass(cls):
         stdout = subprocess.check_output(['git', 'tag'])
         cls.tags = [t.strip() for t in stdout.decode().splitlines()]
 
-        with open('releases.json', 'r') as f:
-            cls.releases = json.load(f)
-        with open('ci_config.json', 'r') as f:
-            cls.ci_config = json.load(f)
+        try:
+            fn = 'releases.json'
+            with open(fn, 'r') as f:
+                cls.releases = json.load(f)
+            fn = 'ci_config.json'
+            with open(fn, 'r') as f:
+                cls.ci_config = json.load(f)
+        except json.decoder.JSONDecodeError as err:
+            what = "Error in file '%s': %s" % (fn, err)
+        finally:
+            raise RuntimeError(what)
```

Comment:
this is going to result in `RuntimeError` *always* being raised, which doesn't seem like what you want. I think you meant
```suggestion
        except json.decoder.JSONDecodeError as err:
            raise RuntimeError(f'{Error in file "{fn)": {err}') from err
```

Which will throw a new exception from the old one, and otherwise pass. A `finally` block is guaranteed to happen regardless of whether there is an exception, and are useful for cases like:
```python
state = make_state()
try:
    state.do_thing()
finally:
    state.cleanup()
```

Reply:
Thanks for this catch.  I'm normally not developing in python so I'm quite rusty.